examples/nextjs-spa/src/pages/settings.tsx

```diff
@@ -78,17 +72,16 @@ const Settings: NextPageWithLayout = () => {
 
     // If ?flow=.. was in the URL, we fetch it
     if (flowId) {
-      getSettingsFlow(String(flowId || "")).catch(
-        (err: AxiosError) =>
-          err.response?.status === 410 ??
-          createSettingsFlow(String(returnTo || "")),
+      getFlow(String(flowId || "")).catch(() =>
+        createFlow(String(returnTo || "")),
       )
       return
     }
 
     // Otherwise we initialize it
-    createSettingsFlow(returnTo)
-  }, [createSettingsFlow, getSettingsFlow, flowId, returnTo, router.isReady])
+    createFlow(returnTo)
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
I'm not sure if we should disable linter rules like this.
Since the functions are memoized through the `useCallback` function, it should be alright to add them to this dependency list.